batch/batch/aioclient.py

```diff
@@ -367,6 +384,11 @@ def __init__(self, session, url=None, token_file=None, token=None, headers=None)
             self.url + path, params=params, cookies=self._cookies, headers=self._headers)
         return await response.json()
 
+    async def _post_data(self, path, data=None):
+        response = await self._session.post(
+            self.url + path, data=data, cookies=self._cookies, headers=self._headers)
```

Comment:
You're passing many more parameters here and using a slightly different method ([than here](https://github.com/hail-is/hail/pull/6403/files#diff-48d48b7c081100ca3f369054a604fe36L330)), wondering why

Reply:
It should be the same. I copied `_post` and didn't change the function signature except data instead of json.

Reply:
Yep, I'm being stupid